File: sdk/py/mod_depin/router.py
import httpx
class Router:
  def __init__(self,api_key,base_url="https://api.moddepin.xyz"):self._k=api_key;self._u=base_url
  def route(self,spec):
    r=httpx.post(f"{self._u}/v1/route",json=spec,headers={"Authorization":f"Bearer {self._k}"},timeout=10);r.raise_for_status();return r.json()

File: sdk/ts/src/router.ts
export class Router{
  constructor(private k:string,private u='https://api.moddepin.xyz'){}
  async route(s:object):Promise<object[]>{
    const r=await fetch(`${this.u}/v1/route`,{method:'POST',headers:{'Authorization':`Bearer ${this.k}`,'Content-Type':'application/json'},body:JSON.stringify(s)});
    if(!r.ok)throw new Error(`${r.status}`);
    return r.json();
  }
}
